recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -97,12 +97,11 @@ do_install_append_class-native () {
     install -m 0644 ${B}/src/sota_tools/garage-sign/lib/* ${D}${libdir}
 }
 
-PACKAGES =+ " ${PN}-examples ${PN}-host-tools ${PN}-tools ${PN}-secondary "
+PACKAGES =+ "${PN}-examples ${PN}-host-tools ${PN}-tools ${PN}-secondary"
 
 FILES_${PN} = " \
                 ${bindir}/aktualizr \
                 ${bindir}/aktualizr-info \
-                ${bindir}/aktualizr-check-discovery \
```

Comment:
Is this no longer needed? Maybe a separate commit, seems a rather unrelated change.